packages/demand-bloc-payment/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@karhoo/demand-bloc-payment",
-  "version": "17.12.0",
+  "version": "17.13.0",
```

Comment:
package lock files should have been deleted

Reply:
Haven't notice that after master merge